lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -51,9 +51,9 @@
 
 BOUNDS_FOR_ECDF = {
     "air_temperature": (
-        Bounds((-40-ABSOLUTE_ZERO, 50-ABSOLUTE_ZERO), "Kelvin")),
+        Bounds((-140-ABSOLUTE_ZERO, 60-ABSOLUTE_ZERO), "Kelvin")),
     "wind_speed": Bounds((0, 50), "m s^-1"),
-    "wind_speed_of_gust": Bounds((0, 70), "m s^-1"),
+    "wind_speed_of_gust": Bounds((0, 350), "m s^-1"),
```

Comment:
This is 782 mph. That seems excessive for any domain on Earth. Tropical cyclone Olivia gave the fastest ever non-tornado related wind speed recorded at 113m/s. Perhaps 120m/s would be more than sufficient?

Reply:
See plots on ticket.